articles/logic-apps/workflow-definition-language-functions-reference.md

```diff
@@ -2807,15 +2807,11 @@ lastIndexOf('<text>', '<searchText>')
 
 If the string or substring value is empty, the following behavior occurs:
 
-* If the string value is empty, `-1` is returned:
+* If only the string value is empty, `-1` is returned:
 
 * If the string and substring values are both empty, `0` is returned.
 
-* If only the substring value is empty, the greater of the following two values is returned:
-
-  * `0`
-
-  * The length of the string, minus 1.
+* If only the substring value is empty, the length of the string, minus 1 is returned.
```

Comment:
Hi @thygesteffensen, thanks for making this contribution. Just a few tweaks to your edits:

```suggestion
* If only the substring value is empty, the function returns the string length minus 1.
```